openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -28,9 +33,21 @@ export class CourseItemsListingView {
         const hasAssessmentType = this.model.get(this.type ? this.type : 'staff_assessment');
         let link = null;
         if (itemViewEnabled && (!this.type || (this.type && hasAssessmentType))) {
+          const displayValue = this.title || formattedValue;
+          if (esgEnabled) {
+            const id = this.model.get('id');
+            link = $('<a>', {
+              text: displayValue,
+              title: displayValue,
+              href: `${esgUrl}/${id}`,
```

Comment:
We have control of both `esgUrl` and `id` so it is less of an issue, but best practice to use a URL join/create instead of string building here.

Reply:
this code doesn't use node. There isn't path join I can use.